plugin-server/src/worker/ingestion/PoE.md

```diff
@@ -0,0 +1,107 @@
+# Persons on Events
+
+## Background
+
+We're getting ready to make a substantial change to the way [persons](https://posthog.com/docs/data/persons) and [events](https://posthog.com/docs/data/events) work by essentially combining them and adding person ids and properties onto events. This is the way we’ll be querying data for all teams using PostHog in the near future.
+
+Why are we doing this? Firstly, it makes queries significantly faster because we no longer have to join tables to get a result; we can just look up everything in the events table instead. One query in our internal tests showed a 400x increase in speed, though 10x is the most common figure...but this beta will help us understand the improvement in real-world conditions.
+
+Secondly, feedback showed that users weren't able to create queries based on person properties at the time of an event. By putting person properties on events, this becomes the new default, while still enabling you to filter insights based on the latest properties using cohorts.
+
+## Roll-out
+
+We're currently offering the new query experience for Beta testers. As it is in Beta, you may or may not spot a few bugs here and there, please use the in-app bug reporting option and mention that you're a PoE (Persons on Events) beta tester.
+
+Since this isn't fully ready yet we might offer you one of the roll-out stage options from below:
+
+## None: JOIN based queries
+
+As reference here are the two aspects of queries that might change and how they work before PoE:
+
+### 1. Filtering on person properties
+
+Latest person properties are joined in during query time.
+
+### 2. Insights counting unique persons
+
+Distinct_id to Person mapping is joined together at query time, see [docs](https://posthog.com/docs/how-posthog-works/queries#insights-counting-unique-persons).
+
+## PoEv1
```

Comment:
I'm not sure we need this level of detail and unsure we need this part at all. When I mentioned some comms I meant more like a paragraph, although this is great to have internally. From a user perspective though, this is a lot more likely to cause confusion and alarm than anything else. I'd just mention that on the first stage if there are multiple logins some users may be double counted and that's about it. Happy to keep this for ourselves though as it is very useful